terraform/shared/video-player/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>

```

Comment:
@darkdrgn2k made changes to this, please review
@Shrinks99 this and the css are the files you want to make pretty